src/packages/cli/src/cli.ts

```diff
@@ -106,5 +130,5 @@ async function startGanache(err: Error) {
     }
   }
 }
-
+console.log("Starting RPC server");
```

Comment:
Is there any reason not to log more things on start up? Maybe this sort of thing might be annoying/unsightly since we don't currently have a way of setting a log level (I'm sure add `debug` support in "the future").

Reply:
I'm not sure if there's a reason to add more logging *here*. We log more in each of the flavor initializations. I'm not sure if this log is necessary, but I also don't see the harm in having it. A CLI doesn't have to be silent if it doesn't have verbosity settings; usually it's pretty verbose if the settings don't exist.